### PR TITLE
Move the branding to P2 

### PR DESCRIPTION
**Pull request by nkolev92**
## Bug

Fixes: https://github.com/NuGet/Client.Engineering/issues/179
Regression: No  
* Last working version:   
* How are we preventing it in future:   

## Fix

Details: Up

## Testing/Validation

Tests Added: No  
Reason for not adding tests:  
Validation:

### CODE REVIEW

File: build/config.props [hunk 1]
```diff
@@ -19,12 +19,12 @@
     
     <!-- ** Change for each new preview/rtm -->
     <!-- Check the VS schedule and manually enter a preview number here that makes sense. -->
-    <ReleaseLabel Condition=" '$(ReleaseLabel)' == '' ">preview.1</ReleaseLabel>
+    <ReleaseLabel Condition=" '$(ReleaseLabel)' == '' ">preview.2</ReleaseLabel>
```
Comment (cristinamanum) [inline build/config.props:22]: When these values need to be increased? How is it tracked that the updates of these values is not missed?

Comment (cristinamanum) [reply build/config.props:22]: Are there partner teams that need to be aware of these changes?

Comment (zivkan) [reply build/config.props:22]: The dates are on the release schedule. Partners have access to the same schedule.

**Review by zivkan — APPROVED** [review]

File: build/config.props (hunk 1 above)
Comment (nkolev92) [reply build/config.props:22]: No partner teams need to be aware of this, it's based on a schedule.
1. In VS, we insert automatically, everyone just picks it up. 
1. In .NET Core we insert automatically and the tooling picks it up. 

As far as ensuring that we don't forget it...we have issue templates for that, like https://github.com/NuGet/Client.Engineering/issues/179 for example.

**Comment by nkolev92** [comment]
Added https://github.com/NuGet/Client.Engineering/issues/179 as an issue to be closed by the merge as this is the last item left on the list.

**Pull request merged**